assets/js/src/subscribers/importExport/import/step_method_selection/method_upload.jsx

```diff
@@ -1,21 +1,69 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import MailPoet from 'mailpoet';
+import ReactStringReplace from 'react-string-replace';
+
+const kbLink = 'http://docs.mailpoet.com/article/126-importing-subscribers-with-csv-files'
```

Comment:
We can update to https://kb.mailpoet.com/article/126-importing-subscribers-with-csv-files